tests/devices/test_default_qubit_tf.py

```diff
@@ -926,6 +928,69 @@ def circuit():
         expected = np.array([amplitude, 0, np.conj(amplitude), 0])
         assert np.allclose(state, expected, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize("theta", [0.5432, -0.232])
+    @pytest.mark.parametrize("op,func", single_qubit_param)
+    def test_one_qubit_param_gates(self, theta, op, func, init_state, tol):
+        """Test the integration of the one-qubit single parameter rotations by passing
+        a TF data structure as a parameter"""
+        dev = qml.device("default.qubit.tf", wires=1)
+        state = init_state(1)
+
+        @qml.qnode(dev, interface='tf')
+        def circuit(params):
+            qml.QubitStateVector(state, wires=[0])
+            op(params[0], wires=[0])
+            return qml.expval(qml.PauliZ(0))
+
+        # Pass a TF Variable to the qfunc

```

Comment:
Worth having tests like these along with the tests in `TestApply`, as the edge case that was missed involved integration with the QNode. Had one unit test for `CRot` previously that passed well.